### PR TITLE
miniupnpd: Fix build on Chimera linux

### PR DESCRIPTION
**Pull request by rbgarga**
Chimera linux is a linux distribution with musl libc and BSD userland.  This series of commits make small adjustments to let miniupnpd to build on it.

### CODE REVIEW

File: miniupnpd/Makefile.linux_nft [hunk 1]
```diff
@@ -130,12 +130,12 @@ install:	miniupnpd $(SRCDIR)/miniupnpd.8 $(SRCDIR)/miniupnpd.conf \
 	$(INSTALL) $(SRCDIR)/netfilter_nft/scripts/nft_flush.sh $(DESTDIR)$(ETCINSTALLDIR)
 	$(INSTALL) $(SRCDIR)/netfilter_nft/scripts/nft_delete_chain.sh $(DESTDIR)$(ETCINSTALLDIR)
 	$(INSTALL) $(SRCDIR)/netfilter_nft/scripts/miniupnpd_functions.sh $(DESTDIR)$(ETCINSTALLDIR)
-	$(INSTALL) --mode=0644 -b $(SRCDIR)/miniupnpd.conf $(DESTDIR)$(ETCINSTALLDIR)
-	sed -i -e "s/^uuid=[-0-9a-f]*/uuid=$(UUID)/" $(DESTDIR)$(ETCINSTALLDIR)/miniupnpd.conf
+	$(INSTALL) -m 0644 -b $(SRCDIR)/miniupnpd.conf $(DESTDIR)$(ETCINSTALLDIR)
+	sed -i'' -e "s/^uuid=[-0-9a-f]*/uuid=$(UUID)/" $(DESTDIR)$(ETCINSTALLDIR)/miniupnpd.conf
```
Comment (miniupnp) [inline miniupnpd/Makefile.linux_nft:134]: are you sure it works as well with gsed ?

Comment (rbgarga) [reply miniupnpd/Makefile.linux_nft:134]: Yes, my first version was using `-i ''` and that broke gsed, but `-i''` without space works fine

```shell
$ echo foo > test
$ gsed -i'' -e 's/foo/bar/' test
$ cat test
bar
```

**Comment by miniupnp** [comment]
@rbgarga Sorry I don't understand the back and forth with commit 066c27054d3a4b4dd71e0cc2236d11deb262068f
Does Chimera linux provides `lsb_release`  ?

**Comment by rbgarga** [comment]
> @rbgarga Sorry I don't understand the back and forth with commit [066c270](https://github.com/miniupnp/miniupnp/commit/066c27054d3a4b4dd71e0cc2236d11deb262068f) Does Chimera linux provide `lsb_release` ?

I just removed it after comment https://github.com/miniupnp/miniupnp/pull/755#issuecomment-2165090352

Yes, Chimera linux provides `lsb_release`

**Pull request merged**